release/release_logs/2.0.0/microbenchmark.json

```diff
@@ -1,274 +1,274 @@
 {
     "1_1_actor_calls_async": [
-        6146.424440513006,
-        128.76132098689908
+        5631.483382626066,
```

Comment:
While this value was still 10% lower - but the preset dataset shows high value of variance, and runs on the PR fixed has shown 6.2k values: https://buildkite.com/ray-project/release-tests-pr/builds/19147#018411d9-6d7f-413c-bb9b-0ef6dacf2186